### ISSUE TITLE
Dynamically set timer value

### ISSUE THREAD

**Issue by kuanghaochina** [spring-projects/spring-statemachine#339]
Need a help. 
The question is that is there any ways to dynamically set or change the value in timerOnce for a internal transition?

**Comment by jvalkeal** [comment]
Nope, not right now. I've been checking if we could use expressions to define time dynamically but uml spec is a bit vague about it.

**Comment by kuanghaochina** [comment]
@jvalkeal Thanks a lot. 👍

**Comment by jvalkeal** [comment]
Lets re-open and make this enhancement request.

**Comment by shiboys** [comment]
Has this question been resolved ?

**Comment by AbstractAlao** [comment]
So has this never been released?
